crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -161,7 +207,83 @@ async fn invalid_nonce_failure_causes_tx_resubmission_under_different_nonce() {
 }
 
 #[tokio::test]
-async fn single_rollup_tx_payload_integrity() {
+async fn invalid_nonce_failure_causes_tx_resubmission_under_different_nonce_grpc_collector() {
+    use crate::helper::mock_sequencer::mount_abci_query_mock;
+
+    // Spawn a composer with a mock sequencer and a mock rollup node
+    // Initial nonce is 0
+    let rollup_id = RollupId::from_unhashed_bytes("test1");
+    let test_composer = spawn_composer(&[rollup_id.to_string().as_str()]).await;
+    tokio::time::timeout(
+        Duration::from_millis(100),
+        test_composer.setup_guard.wait_until_satisfied(),
+    )
+    .await
+    .expect("setup guard failed");
```

Comment:
Maybe you copied it from another test - but this `expect` message (like above) kinda states the obvious.